containers/java-postgres/test-project/src/test/java/com/mycompany/app/AppTest.java

```diff
@@ -0,0 +1,44 @@
+/*--------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+package com.mycompany.app;
+
+import org.junit.Test;
+
+import junit.framework.AssertionFailedError;
+
+import static org.junit.Assert.*;
+
+import java.io.IOException;
+import java.net.UnknownHostException;
+
+public class AppTest
+{
+    App newApp = new App();
+    public AppTest() {
+    }
+
+    @Test
+    public void testApp()
```

Comment:
nit: Are these necessary ? Seems like it is not testing anything

Reply:
that was an existing @Test from the `java` folder that we left in place.

Reply:
Technically you don't need `main` at all in the `App.java` file. Just having a class with the static method is good enough.

Reply:
I agree with you, but again, we're `main`-ly just following the templates that already exist. Most of our changes are additive.